privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -30,19 +29,19 @@ <h2>{% ftlmsg 'onboarding-premium-subheadline-html' %}</h2>
         <li class="c-premium-onboarding-step c-premium-onboarding-step-2 {% if user_profile.onboarding_state == 1 %} is-visible {% endif %}">
             <div class="mzp-l-content mzp-t-content-xl">
                 <div class="c-premium-onboarding-headlines">
-                    <h1>Custom domain for sharing aliases</h1>
-                    <h2>Now you can control what hits your inbox, one email at a time.</h2>
+                    <h1>{% ftlmsg 'multi-part-onboarding-premium-domain-headline' %}</h1>
+                    <h2>{% ftlmsg 'multi-part-onboarding-premium-domain-subheadline' %}</h2>
                 </div>
 
                 <div class="c-premium-onboarding-content">
                     <div class="c-premium-onboarding-content-image">
                         <img src="{% static 'images/dashboard-onboarding/man-laptop-email-alt.svg' %}" alt="Man sitting with laptop computer">
                     </div>
                     <div class="c-premium-onboarding-content-description">
-                        <p><strong>Use a custom domain for sharing aliases:</strong> With a custom domain, you can make your “newsletter” alias different from your “shopping” alias. </p>
+                        <p><strong>{% ftlmsg 'multi-part-onboarding-premium-domain-title' %}</strong>{% ftlmsg 'multi-part-onboarding-premium-domain-description' %}</p>
```

Comment:
Q: Should this have a space after the `</strong>`?

Reply:
```suggestion
                        <p><strong>{% ftlmsg 'multi-part-onboarding-premium-domain-title' %}</strong> {% ftlmsg 'multi-part-onboarding-premium-domain-description' %}</p>
```

Reply:
Yes it should! :)